File: PastaPHP/Model/Professor.php
<?php

require_once __DIR__. "/Pessoa.php";

class Professor extends UsuarioSistema {
   

    public function __construct($nome,$id, $cpf, $email,$senha){
        parent:: __construct($nome, $id, $cpf, $email, $senha);
    }

        public function __toString() {
        return parent::__toString() . " Professor:";
    }

    }
?>

File: PastaJava/Model/Professor.java
package PastaJava.model;

public class Professor extends UsuarioSistema {
    public Professor() {}

    public Professor(String nome, int id, String cpf, String email, String senha) {
        super(nome, id, cpf, email, senha);
    }

    @Override
    public String toString() {
        return super.toString() + " Professor: " ;
    }
}
